subsys/mpsl/cx/Kconfig

```diff
@@ -23,13 +21,23 @@ choice MPSL_CX_CHOICE
 config MPSL_CX_THREAD
 	select NRFX_GPIOTE
 	select GPIO
-	bool "Thread Radio Coexistence"
+	bool "Thread Radio Coexistence [EXPERIMENTAL]"
```

Comment:
All experimental features should now `select EXPERIMENTAL`, now that we have an sdk-zephyr patch which includes `CONFIG_EXPERIMENTAL` pulled from upstream by the latest upmerge.

(I'm including this comment here since there is a requirement to update sdk-zephyr to get access to the `coex` property in the radio node, and that sdk-zephyr update will be on top of the latest upmerge.)

Reply:
great, didn't know it had been merged

Reply:
alright, I tried it and got a checkpatch issue, then grepped a bit in zephyr and it looks like it's not in yet.

Reply:
Sorry about that, I could have sworn this was in the latest upmerge already.